azure-pipelines.yml

```diff
@@ -223,8 +232,8 @@ jobs:
       SessionTimeout: '60'
       MaxConcurrency: '50'
       MaxRetryAttempts: '2'
-      condition: eq(variables['SignFiles'], 'true')
-  
+    condition: and(succeeded(), eq(variables['SignFiles'], 'true'))
```

Comment:
The indentation was wrong on these conditions, leading to them being totally ignored.
They were also missing the `succeeded()` condition, which means that signing would have been attempted even if the pipeline had already failed.